chart/flux/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.11.0"
+appVersion: "1.11.1"
 version: 0.7.0
```

Comment:
```suggestion
version: 0.7.1
```

Reply:
@timfpark you've disabled the push access to your branch. Can you commit the above?